docs/references/cmcspec.mdx

```diff
@@ -0,0 +1,552 @@
+import { MarkdownChipRow } from "/src/components/Chip/MarkdownChipRow";
+
+# Cycles Minting Canister (CMC) API
+
+<MarkdownChipRow labels={["Reference"]} />
+
+The Cycles Minting Canister (CMC) is a system canister on the Internet Computer responsible for converting ICP tokens into **cycles**, tracking the total cycles minted, and providing configuration and exchange rate data. It supports:
+
+-   Topping up existing canisters.
+-   Creating new canisters.
+-   Minting and depositing minted cycles into cycle ledger-managed accounts.
+-   Fetching exchange rates and subnet configuration.
```

Comment:
here I was wondering: is this exchange rate just between cycles:ICP or more? (I thought it used to be more but now we have a separate exchange rate canister. But since I am also not sure, I wonder if others might have the same question :)